met-web/src/components/layout/SideNav/SideNav.tsx

```diff
@@ -28,43 +75,118 @@ const DrawerBox = () => {
+                    <ListItemButton
+                        disableRipple
+                        sx={{
+                            '&:hover, &:active, &:focus': {
+                                backgroundColor: 'transparent',
+                            },
+                        }}
+                        data-testid={`SideNav/${route.name}-button`}
+                        onClick={() => {
+                            navigate(route.path);
+                            setOpen(false);
+                        }}
+                    >
+                        {handleFontAwesomeIcon(route, itemType)}
+                        <MetHeader4
+                            style={{
+                                color: 'selected' === itemType ? Palette.primary.light : Palette.text.primary,
+                                fontWeight: 'selected' === itemType ? 'bold' : '500',
+                                fontSize: '1rem',
+                            }}
+                        >
+                            {route.name}
+                            <When condition={currentBaseRoute === route.base}>
+                                <span style={{ position: 'absolute', right: '1rem' }}>
+                                    <FontAwesomeIcon icon={faCheck} />
+                                </span>
+                            </When>
+                        </MetHeader4>
+                    </ListItemButton>
+                </ListItem>
+                <Divider sx={{ backgroundColor: colors.surface.gray[30] }} />
+                <When condition={'User Admin' === route.name}>
+                    <UserGuideNav />
+                </When>
+            </>
+        );
+    };
+
+    const handleFontAwesomeIcon = (route: Route, itemType: string) => {
+        const fontAwesomeStyles = {
+            fontSize: '1.1rem',
+            color: 'selected' === itemType ? Palette.primary.light : Palette.text.primary,
+            paddingRight: '0.75rem',
+            width: '1.1rem',
+        };
+        const iconAssignments: IconAssignments = {
+            Home: faHouse,
+            Engagements: faPeopleArrows,
+            Surveys: faSquarePollHorizontal,
+            Metadata: faTags,
+            Languages: faGlobe,
+            'User Admin': faUserGear,
+            'Tenant Admin': faHouseUser,
+            'MET Feedback': faMessagePen,
+        };
+        // If the route name doesn't match an expected page title, return the broken link icon
+        const possiblePageTitles: string[] = Object.keys(iconAssignments).filter((key) => key.includes(route.name));
+        if (0 === possiblePageTitles.length) {
+            return <FontAwesomeIcon icon={faLinkSlash} style={fontAwesomeStyles} />;
+        }
+        // Otherwise return the icon for the icon assignments key name that matches the route name.
+        return <FontAwesomeIcon icon={iconAssignments[route.name]} style={fontAwesomeStyles} />;
+    };
+
     return (
         <Box
```

Comment:
Nice making this menu a list in HTML! Since this is a [page region](https://www.w3.org/WAI/tutorials/page-structure/regions/), assistive technologies can benefit from having it marked as such. The below code will make the outer element a `<nav>` and give it a label.

https://www.w3.org/WAI/tutorials/page-structure/regions/#navigation

```suggestion
        <Box
            component="nav"
            aria-labelledby="Editor navigation"
```

Reply:
Aria-labelledby is for passing in an element ID; you want aria-label instead

Reply:
Thanks @NatSquared that's my bad. Also I think the label itself could probably be improved to something like "Admin navigation menu"